README.rst

```diff
@@ -110,7 +115,7 @@ Now you're ready to start your PyGotham server::
 
 You should see the PyGotham site at::
 
-    http://0.0.0.0:5000/<slug>/
+    http://<slug>.0.0.0.0:5000/
```

Comment:
Nice catch. There were two errors there:
- The server name should not actually include the `slug`.
- As you pointed out, if `SERVER_NAME` is set, the URL must match.
Both fixed and working now.